src/telemetry/generated/appEvents.js

```diff
@@ -0,0 +1,39 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// AUTOGENERATED BY glean_parser v8.1.1. DO NOT EDIT. DO NOT COMMIT.
+
+import EventMetricType from "@mozilla/glean/private/metrics/event";
+
+/**
+ * A generic button click event with extras.
+ *
+ * Generated from `app_events.button_click`.
+ */
+export const buttonClick = new EventMetricType(
+  {
+    category: "app_events",
+    name: "button_click",
+    sendInPings: ["events"],
+    lifetime: "ping",
+    disabled: false,
+  },
+  ["label"]
+);
+
+/**
+ * A generic page view event with extras.
+ *
+ * Generated from `app_events.page_view`.
+ */
+export const pageView = new EventMetricType(
+  {
+    category: "app_events",
+    name: "page_view",
+    sendInPings: ["events"],
+    lifetime: "ping",
+    disabled: false,
+  },
+  ["label"]
```

Comment:
Not sure if this is the right place for a feature request, but it would be great if the type definition of `ExtraMap` / `ExtraValues` could be derived from the values passed here, so that we can get autocompletion when we pass an argument to `appEvents.pageView.record`.

Reply:
@Vinnl would you kindly file a bug in bugzilla's Glean SDK component? It's a totally reasonable request

Reply:
Sure! https://bugzilla.mozilla.org/show_bug.cgi?id=1850542

Note that I did not know which version to specify - the lockfile says that it's `2.0.1`, which was not an option in Bugzilla, so I set it to `unspecified`.

Reply:
@Vinnl that's perfect, thank you for filing!